pkg/innerring/innerring.go

```diff
@@ -388,6 +394,48 @@ func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper) (*Server, error
 		return nil, err
 	}
 
+	cnrClient, err := invoke.NewNoFeeContainerClient(server.morphClient, server.contracts.container)
```

Comment:
I would rearrange code a bit and move all morph client wrapper constructors in one place, e.g. somewhere above together with audit client wrapper.

```go
	// create mainnet client
	server.mainnetClient, err = createClient(ctx, mainnetChain)
	if err != nil {
		return nil, err
	}

	// create morph client wrappers
 	auditClient, err = invoke.NewNoFeeAuditClient(server.morphClient, server.contracts.audit)
 	...
 	cnrClient, err := invoke.NewNoFeeContainerClient(server.morphClient, server.contracts.container)
 	...
 	nmClient, err := invoke.NewNoFeeNetmapClient(server.morphClient, server.contracts.netmap) 
 	...
	balClient, err := invoke.NewNoFeeBalanceClient(server.morphClient, server.contracts.balance)
```

Reply:
I would perform code improvements in a separate PRs (e.g. pre-release) if you do not mind @alexvanin.